src/components/Form/Form.js

```diff
@@ -117,23 +69,22 @@ export default function Form() {
 
 				<StyledButton type="submit">Add to diary</StyledButton>
 			</form>
-			<DiaryHeadline />
-			<StyledWrapper>
-				{entries.map(entry => {
-					return (
-						<StyledCard key={entry.id}>
-							<Date />
-							<StyledH4>You were grateful for:</StyledH4>
-
-							<ul>
-								<StyledLi key={entry.id}>{entry.firstEntry}</StyledLi>
-								<StyledLi key={entry.id}>{entry.secondEntry}</StyledLi>
-								<StyledLi key={entry.id}>{entry.thirdEntry}</StyledLi>
-							</ul>
-						</StyledCard>
-					);
-				})}
-			</StyledWrapper>
-		</>
+		</DynamicWrapper>
 	);
 }
+
+const StyledInput = styled.textarea`
```

Comment:
I would rename this styled component as this is a textarea and not a simple input field.
```suggestion
const StyledTextarea = styled.textarea`
```